ansible/roles/connection_db/tasks/stop_db.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Stop servercfgd
+  systemd:
+    name: servercfgd
+    state: stopped
+
+- name: Ensure Redis is stopped
+  service:
+    name: redis-server
+    state: stopped
+
+- name: Remove Redis
+  apt:
+    name: redis-server
+    state: absent
+    purge: yes
+
+- name: Remove py-redis
```

Comment:
Yes, like dynamically allocated vlan ids.
But for stage one, `connection_db` is just a copy of connection graph files. So this will be considered when dynamic changes to `connection_db` will be introduced in the future.